go.mod

```diff
@@ -1,10 +1,16 @@
 module github.com/projectdiscovery/goconfig
 
-go 1.14
+go 1.18
 
 require (
 	github.com/projectdiscovery/reflectutil v0.0.0-20210804085554-4d90952bf92f
```

Comment:
Not mentioned in the issue - anyway I catch the occasion to suggest changing to:
```
reflectutil "github.com/projectdiscovery/utils/reflect"
```